packages/devtools_app/lib/src/debugger/debugger_controller.dart

```diff
@@ -732,6 +733,70 @@ class DebuggerController extends DisposableController
     return boundsVariables.map((bv) => Variable.create(bv)).toList();
   }
 
+  // Decodes the bytes into the correctly sized values based on the instance
+  // type, falling back to raw bytes. The switch is based on the allowed
+  // values of `instance.bytes`.
+  List<Variable> _createVariablesForBytes(Instance instance) {
+    final bytes = base64.decode(instance.bytes);
+    final boundVariables = <BoundVariable>[];
+    List<dynamic> result;
+    switch (instance.kind) {
```

Comment:
I'm not in love with the giant switch, but it seemed like the simplest way to accomplish this.